test/integration/targets/inventory_kubevirt_conformance/runme.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -eux
+
+source virtualenv.sh
+
+virtualenv --system-site-packages --python "${ANSIBLE_TEST_PYTHON_INTERPRETER:-python}" "${OUTPUT_DIR}/kubevirt-inventory"
+source "${OUTPUT_DIR}/kubevirt-inventory/bin/activate"
```

Comment:
With `source virtualenv.sh` these two lines for creating and activating the virtualenv are no longer needed.

Reply:
done